spec/models/claims/totals_spec.rb

```diff
@@ -1,114 +1,116 @@
-  subject { create(:claim) }
-  let(:expenses) { [3.5, 1.0, 142.0].each { |amount| create(:expense, claim_id: subject.id, amount: amount) } }
+  subject(:claim) { create(:claim) }
+  let(:expenses) { [3.5, 1.0, 142.0].each { |amount| create(:expense, claim_id: claim.id, amount: amount) } }
   let(:fee_type) { create(:fixed_fee_type) }
 
   context 'fees total' do
-
     describe '#calculate_fees_total' do
       it 'calculates the fees total' do
-        expect(subject.calculate_fees_total).to eq(25.0)
+        expect(claim.calculate_fees_total).to eq(25.0)
       end
     end
 
     describe '#update_fees_total' do
       it 'stores the fees total' do
-        expect(subject.fees_total).to eq(25.0)
+        expect(claim.fees_total).to eq(25.0)
       end
 
       it 'updates the fees total' do
-        create(:fixed_fee, fee_type: fee_type, claim_id: subject.id, rate: 2.00)
-        subject.reload
-        expect(subject.fees_total).to eq(27.0)
+        create(:fixed_fee, fee_type: fee_type, claim_id: claim.id, rate: 2.00)
+        claim.reload
+        expect(claim.fees_total).to eq(27.0)
       end
 
       it 'updates total when claim fee destroyed' do
-        create(:fixed_fee, fee_type: fee_type, claim_id: subject.id, rate: 2.00)
-        subject.fees.first.destroy
-        subject.reload
-        expect(subject.fees_total).to eq(2.0)
+        create(:fixed_fee, fee_type: fee_type, claim_id: claim.id, rate: 2.00)
+        claim.fees.first.destroy
+        claim.reload
+        expect(claim.fees_total).to eq(2.0)
       end
     end
   end
 
   context 'expenses total' do
-    before { expenses; subject.reload }
+    before { expenses; claim.reload }
 
     describe '#calculate_expenses_total' do
       it 'calculates expenses total' do
-        expect(subject.calculate_expenses_total).to eq(146.5)
+        expect(claim.calculate_expenses_total).to eq(146.5)
       end
     end
 
     describe '#update_expenses_total' do
       it 'stores the expenses total' do
-        expect(subject.expenses_total).to eq(146.5)
+        expect(claim.expenses_total).to eq(146.5)
       end
 
       it 'updates the expenses total' do
-        create(:expense, claim_id: subject.id, amount: 3)
-        subject.reload
-        expect(subject.expenses_total).to eq(149.5)
+        create(:expense, claim_id: claim.id, amount: 3)
+        claim.reload
+        expect(claim.expenses_total).to eq(149.5)
       end
 
       it 'updates expenses total when expense destroyed' do
-        subject.expenses.first.destroy
-        subject.reload
-        expect(subject.expenses_total).to eq(143.0)
+        claim.expenses.first.destroy
+        claim.reload
+        expect(claim.expenses_total).to eq(143.0)
       end
     end
   end
 
   context 'expenses vat' do
-    let!(:expenses) { [3.5, 1.0, 142.0].each { |amount| create(:expense, claim_id: subject.id, amount: amount, vat_amount: 2) } }
-
-    before { subject.reload }
+    before do
+      VatRate.delete_all
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2271/files#r173426782)